docs/starters.yml

```
@@ -1231,3 +1231,20 @@
     - Bootstrap v4 support
     - Css Modules support
     - ESLint, Prettier
+- url: https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/
+  repo: https://github.com/DanShai/gatsbyv2-scientific-blog-machine-learning
+  description: Machine learning ready and scientific blog starter
+  tags:
+    - Blog
+    - Katex
+    - Tensorflow
+    - Csv
+    - Graph mermaid
```

```suggestion
    - Graph
```

I think that would be better fitting and more general